src/v2/Components/Modal/Modal.tsx

```diff
@@ -48,6 +49,9 @@ export class Modal extends React.Component<ModalProps> {
 
   close = () => {
     this.props.onClose()
+    if (this.props.hideCloseButton) {
```

Comment:
The naming of `hideCloseButton` feels a bit indirect for the action it's taking. Unless i'm misunderstanding?

Reply:
Totally @damassi. We lifted the name from palette's dialog but then actually added close button back and forgot to change the name. Naming is tricky. What would convey the concept of `prevent close without taking action`. Maybe that can be the name. Too long though:)

Reply:
Yeah, I still need to think about that, and whether we should also allow users to close the modal by clicking outside or pressing `esc`.

I'm leaning towards allowing users to press `esc`, but not click outside to close the modal. The difference is that clicking the X or pressing `esc` feels like a direct action that the user is taking, whereas clicking outside of the modal feel indirect and likely to cause confusion. I'd love to hear your thoughts about the UX of that.

Reply:
I think the UX is just fine for something like this. We very explicitly want the user to take an action to proceed, versus the logged out experience on /artist/id where we show the login form after scroll, and they can choose to dismiss to continue scrolling. 

Naming-wise, something explicit like `disableCloseOnBackgroundClick` (or some such thing) would communicate it more clearly.